README.md

```diff
@@ -1,24 +1,180 @@
-# README
+<!-- TABLE OF CONTENTS -->
```

Comment:
# Frontend re-review

- [ ] There is still some commented code on `App.js`:

![image](https://github.com/dheerajsachdeva/Book-An-Appointment-Backend/assets/57421823/0701fcc0-c17a-4d45-b93d-d51da307673b)

Kindly remove it.

- [ ] Good job adding tests!  However, they are not passing on my end:


![image](https://github.com/dheerajsachdeva/Book-An-Appointment-Backend/assets/57421823/61998dcd-9de3-4a6b-9d30-4e180798652c)

It seems that Jest is failing to parse the tests correctly:

![image](https://github.com/dheerajsachdeva/Book-An-Appointment-Backend/assets/57421823/589cb804-de0b-49fe-8812-1707c4432129)

Are any prerequisites needed apart from `npm install` in order to run them correctly? If so, kindly include the instructions on your readme.  If this is a bug, kindly fix it so your tests work as intended. 👍